test/e2e/testsuites/metrics.go

```diff
@@ -0,0 +1,207 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testsuites
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/google/uuid"
+	metricspkg "github.com/googlecloudplatform/gcs-fuse-csi-driver/pkg/metrics"
+	"github.com/googlecloudplatform/gcs-fuse-csi-driver/test/e2e/specs"
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	dto "github.com/prometheus/client_model/go"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2evolume "k8s.io/kubernetes/test/e2e/framework/volume"
+	storageframework "k8s.io/kubernetes/test/e2e/storage/framework"
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+var expectedMetricNames = map[string]int{
+	"fs_ops_count":                15,
+	"fs_ops_error_count":          2,
+	"fs_ops_latency":              15,
+	"gcs_download_bytes_count":    1,
+	"gcs_read_count":              1,
+	"gcs_read_bytes_count":        0, // TODO: cannot find this metric in e2e tests.
```

Comment:
Hi @sethiay, @kislaykishore , I modified the [GCSFuse PR](https://github.com/GoogleCloudPlatform/gcsfuse/pull/1859/files#diff-57454978dcd397e797f4c48e4b1f80b4b249655d5fc975ad55c54d560ba01de1) to enable the gcs metrics, then I see some new metrics.

However, I still don't see the metric `gcs_read_bytes_count`. Could you help check what I am missing? Thank you!